cypress/integration/ete-okta/registration.2.cy.ts

```diff
@@ -1109,7 +1109,12 @@ describe('Registration flow', () => {
             cy.contains(emailAddress);
             cy.contains('Continue')
               .should('have.attr', 'href')
-              .and('include', `https://${Cypress.env('BASE_URI')}/consents`);
+              .and(
+                'include',
+                `https://${Cypress.env(
+                  'BASE_URI',
+                )}/signin/refresh?returnUrl=https%3A%2F%2Fprofile.thegulocal.com%2Fconsents`,
```

Comment:
Are these URL changes just extra bugs you found or are they directly as a result of the changes?

Reply:
Oh great 👍